README.md

```diff
@@ -25,6 +25,24 @@ The AutoGPT Arena Hackathon saw [**evo.ninja**](https://github.com/polywrap/evo.
 
 📈 To challenge evo.ninja, AutoGPT, and others, submit your benchmark run to the [Leaderboard](#-leaderboard), and maybe your agent will be up here next!
 
+
+### How to Run Evo Ninja
+To run Evo Ninja, follow these detailed steps:
+1. **Pre-Requisites**: Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [nodejs](https://nodejs.org/en/download/package-manager#alpine-linux), [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable), and [nvm](https://github.com/nvm-sh/nvm#installing-and-updating).
+2. **Installation**:
+    - Clone the repository: `git clone https://github.com/polywrap/evo.ninja`
+    - Copy and rename the environment template: `cp .env.template .env`
```

Comment:
instead of duplicating this information, I would just point them to the evo.ninja README within the `release/autogpt` branch: https://github.com/polywrap/evo.ninja/tree/release/autogpt#installation

Reply:
Made it so all the instructions arent duplicated, source of truth will be in the branch you mentioned